components/emoji_picker/utils/index.test.ts

```diff
@@ -211,36 +213,19 @@ describe('getFilteredEmojis', () => {
         expect(getFilteredEmojis(allEmojis as any, filter, recentEmojisString, userSkinTone)).toEqual(filteredResults);
     });
 
-    test('Should filter recent emojis containning skin tone with user skin tone', () => {
+    test('Should filter recent emojis', () => {
         const allEmojis = {
             thumbsup: thumbsupEmoji,
             thumbsupDark: thumbsupEmojiDarkSkin,
             thumbsupLight: thumbsupEmojiLightSkin,
             thumbsupMedium: thumbsupEmojiMediumSkin,
         };
         const filter = 'thumbs';
-        const recentEmojisString = ['thumbsupMedium', 'thumbsupDark'];
-        const userSkinTone = SkinTones.Dark;
-
-        const filteredResults = [
-            thumbsupEmojiDarkSkin,
-            thumbsupEmoji,
-        ];
-
-        expect(getFilteredEmojis(allEmojis as any, filter, recentEmojisString, userSkinTone)).toEqual(filteredResults);
-    });
-
-    test('Should filter recent emojis containning skin tone and replace with user skin tone', () => {
```

Comment:
This test case is no longer needed since recent emojis will have been deduped and had their skin tones set before this point